### PR TITLE
Post creation page

### PR DESCRIPTION
**Pull request by ArCheeq**

### CODE REVIEW

File: src/store/slices/creationPostDataSlice.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  description: "",
+  finalAmount: "",
+  donationLink: "",
+  donationCards: "",
+  email: "",
+  phone: "",
+};
+
+const creationPostData = createSlice({
+  name: "postData",
```
Comment (Misha999777) [inline src/store/slices/creationPostDataSlice.js:21]: Let's rename to postCreation

File: src/axios/index.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import axios from "axios"
+
+const _baseUrl = 'https://backend/uploadImages';
+
+
+export const uploadImagesInstance = axios.create({
+    baseURL: _baseUrl,
+    timeout: 5000,
```
Comment (Misha999777) [inline src/axios/index.js:8]: 5 seconds will not be enough to upload images in real life

**Review by Misha999777 — APPROVED** [review]
LGTM, except for minor comments

File: src/axios/index.js (hunk 2 above)
Comment (ArCheeq) [reply src/axios/index.js:8]: Will 2 minutes be enough?

File: src/components/Uploader.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+function Uploader() {
+  const [hovered, setHovered] = useState(false);
+  const inputRef = useRef(null);
+
+  const dispatch = useDispatch();
+
+  const [uploadImages, loading, error] = useImageUploader();
+
+  // Variable is used to create dashed border with large line spacing, please don't change !
+  const _formBorder = `url("data:image/svg+xml,%3csvg width='100%25' height='100%25' xmlns='http://www.w3.org/2000/svg'%3e%3crect width='100%25' height='100%25' fill='none' rx='24' ry='24' stroke='${
```
[the remark this answers is not in the document]
Comment (ArCheeq) [reply src/components/Uploader.js:19]: ok, I'll fix it

**Pull request merged**